lib/sequent/core/helpers/type_conversion_support.rb

```diff
@@ -0,0 +1,27 @@
+require 'active_model'
+
+module Sequent
+  module Core
+    module Helpers
+      # Will parse all values to the correct types.
+      # The raw values are typically posted from the web and are therefor mostly strings.
+      # To parse a raw value your class must have a parse_from_string method that returns the parsed values.
+      # See sequent/core/ext for currently supported classes.
+      module TypeConversionSupport
+        def parse_attrs_to_correct_types!
```

Comment:
I expect a `Command` to be immutable. Thus, I would prefer to rename `parse_attrs_to_correct_types!` to `parse_attrs_to_correct_types` and let it return a copy of the command with it's values (read: types) changed.